lib/slim_lint/linter/trailing_blank_lines.rb

```diff
@@ -0,0 +1,18 @@
+module SlimLint
+  # Checks for trailing whitespace.
+  class Linter::TrailingBlankLines < Linter
+    include LinterRegistry
+
+    on_start do |_sexp|
+      dummy_node = Struct.new(:line)
+
+      if !document.source.end_with?("\n")
+        report_lint(dummy_node.new(document.source_lines.size),
+                    'No blank line in the end of file')
+      elsif document.source.lines.last.blank?
```

Comment:
`document.source.lines` should use `document.source_lines` so you aren't re-splitting the string for newlines here and on line 13 below.